packages/components/src/styles/interactive/interactive.stories.ts

```diff
@@ -64,88 +36,33 @@ export const Default = {
-      args
-    });
-  }
-};
-
-/**
- * Use the `inverted` class to invert the colors of an interactive element.
+ * Use the `&--inverted` class when displayed on primary background.
+ *
+ * On darker backgrounds text/white inverted colour will substitute primary.
  */
 
 export const Inverted = {
-  parameters: { controls: { exclude: ['sd-interactive--inverted', 'sd-interactive--reset'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: [{ type: 'attribute', name: 'sd-interactive--inverted', values: [false, true] }]
-      },
-      options: {
-        templateContent: '<button class="%CLASSES%">%SLOT%</button>',
-        templateBackgrounds: { alternate: 'y', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
-      },
-      constants: { type: 'attribute', name: 'sd-interactive--inverted', value: true },
-      args
-    });
-  }
+  render: () => html`
+    <div class="bg-primary p-4">
+      <button class="sd-interactive sd-interactive--inverted sd-interactive--reset">Inverted</button>
+    </div>
+  `
 };
 
 /**
- * Use the `reset` class to reset the default browser styles of e. g. a button.
+ * Use the `sd-interactive--disabled` class to disabled an interactive element.
+ *
+ * This works as well when setting an `disabled` attribute on the element.
  */
 
-export const Reset = {
-  parameters: { controls: { exclude: ['sd-interactive--reset'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: [{ type: 'attribute', name: 'sd-interactive--reset', values: [false, true] }]
-      },
-      options: { templateContent: '<button class="%CLASSES%">%SLOT%</button>' },
-      constants: { type: 'attribute', name: 'sd-interactive--reset', value: true },
-      args
-    });
-  }
+export const Disabled = {
+  render: () => html` <button class="sd-interactive sd-interactive--disabled sd-interactive--reset">Disabled</button> `
 };
 
 /**
- * Examples: This shows how sd-interactive looks in different contexts.
+ * Use the `sd-interactive--reset` class to reset the default browser styles of e. g. a button.
  */
 
-export const Examples = {
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'slot',
-          name: 'default',
-          values: [
-            'Text',
-            '<span>Text</span>',
-            '<sd-icon library="global-resources" name="system/picture"></sd-icon>'
-          ].map(content => {
-            return {
-              value: content,
-              title: content.replace('library="global-resources" name="system/picture"', '...')
-            };
-          }),
-          title: 'content'
-        }
-      },
-      options: { templateContent: '<button class="%CLASSES%">%SLOT%</button>' },
-      args
-    });
-  }
+export const Reset = {
+  render: () => html` <button class="sd-interactive sd-interactive--reset">Reset</button> `
```

Comment:
```suggestion
  render: () => html`
 <style>
        button.sd-interactive:not(.sd-interactive--reset) {
          background-color: rgb(239, 239, 239);
          border: 1px solid rgb(204, 204, 204);
          border-radius: 4px;
          color: rgb(51, 51, 51);
          cursor: pointer;
          font-family: 'Roboto', sans-serif;
          font-size: 14px;
          font-weight: 500;
          line-height: 20px;
          padding: 8px 16px;
        }
        button.sd-interactive:not(.sd-interactive--reset):hover {
          background-color: rgb(230, 230, 230);
          border-color: rgb(204, 204, 204);
          color: rgb(51, 51, 51) !important;
        }
        button.sd-interactive:not(.sd-interactive--reset):active {
          background-color: rgb(204, 204, 204);
        }
      </style>
      <div class="flex gap-4">
        <button class="sd-interactive">Lorem Ipsum</button>
        <button class="sd-interactive sd-interactive--reset">Lorem Ipsum</button>
      </div>`
```

Looking at it now, I think it makes a lot more sense to include a button without the reset, and one button with reset. Since the big reason we decided to remove it was because it wasn't working, I did a quick troubleshooting to save you time.

Basically our old stories has some extra styles to showcase the difference and I added them to this story. Please let me know if you have any questions :)

Reply:
Updated